quinn-proto/src/connection/mod.rs

```diff
@@ -1493,7 +1479,7 @@ where
             return;
         }
 
-        if self.anti_amplification_blocked() {
+        if self.path.anti_amplification_blocked(self.mtu) {
```

Comment:
Hmm, does `mtu` also belong in `self.path`?

Reply:
I think so. However, I'm leaving the argument in place because I anticipate it being immediately useful in #953.